servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/ContextPreservingSingleSubscriber.java

```diff
@@ -19,11 +19,9 @@
 import io.servicetalk.concurrent.SingleSource;
 import io.servicetalk.concurrent.SingleSource.Subscriber;
 import io.servicetalk.context.api.ContextMap;
-import io.servicetalk.context.api.ContextMapHolder;
 
 import javax.annotation.Nullable;
 
-import static io.servicetalk.concurrent.api.AsyncContextMapThreadLocal.CONTEXT_THREAD_LOCAL;
 import static java.util.Objects.requireNonNull;
 
 class ContextPreservingSingleSubscriber<T> implements Subscriber<T> {
```

Comment:
Not necessary for now, but whenever you have time could you please verify that we indeed call `AsyncContextProvider.wrapSingleSubscriber(...)` and similar wrappers for Completable/Publisher in all operators whey it's necessary. Would be great to make sure we don't miss context propagation.